intro/classes/index.html

```diff
@@ -55,8 +55,13 @@ <h2><a id="class-intro" class="js"></a>What Are Classes and Why Do We Care?</h2>
         </p>
         <p>
             A "class" is ultimately just a container for some data and functionality. The key part to recognize about a "class"
-            is that the "class" term usually specifically refers to the abstract container itself. So <code>int</code>  is a class
-            but <code>1</code> is not. <code>1</code> is what's called an <i>instance</i> of the <code>int</code> class.
+            is that the "class" term usually specifically refers to the abstract container itself. So <code>int</code> is a class
+            but <code>1</code> is not. <code>1</code> is what's called an <i>instance</i> of the <code>int</code> class. Note that
+            my example is not perfect. <code>int</code> behaves a bit differently than most classes you'll define for your own use. But,
```

Comment:
"Note, however, that int behaves ..."? like, in terms of terminology and stuff the example is fine, just that it's unusual to what one would usually do